roles/falcon/tasks/api.yml

```diff
@@ -1,82 +1,57 @@
-
-- name: CrowdStrike Falcon Installer | Authenticate to CrowdStrike API
+---
+- name: CrowdStrike Falcon | Authenticate to CrowdStrike API
   uri:
-    url: https://{{ crowdstrike.api.url }}/oauth2/token
+    url: "https://{{ falcon_api_url }}/oauth2/token"
     method: POST
-    body: ???
-    return_content: yes
+    body_format: json
+    body: 
+      "client_id={{ falcon_api_client_id }}&client_secret={{ falcon_api_client_secret }}"
+    return_content: true
+    status_code: 201
     headers:
         content-type: application/x-www-form-urlencoded
-    register: crowdstrike.oauth2_token
-
-#crowdstrike_oauth2_token=$(curl --request POST \
-#--silent \
-#--url "https://$target_cloud_api/oauth2/token" \
-#--header "content-type: application/x-www-form-urlencoded" \
-#--data client_id=$CS_API_CLIENT_ID \
-#--data client_secret=$CS_API_CLIENT_SECRET | \
-#  python3 -c "import sys, json; print(json.load(sys.stdin)['access_token'])")
-
-- name: CrowdStrike Falcon Installer | API Authorization Output
-  debug:
-    msg: "{{ crowdstrike.oauth2_token.json }}"
```

Comment:
Should we keep the debug?

Reply:
@shawndwells why? No need to print out the OAuth2 Token to output.